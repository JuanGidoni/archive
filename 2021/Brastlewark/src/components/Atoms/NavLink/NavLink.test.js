import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import NavLink from './NavLink';

describe('<NavLink />', () => {
 it('renders correctly and match snapshot', () => {
  const tree = renderer
   .create(<Router><NavLink to="/" text="Example" className="example" /></Router>)
   .toJSON();
  expect(tree).toMatchSnapshot();
 });
});