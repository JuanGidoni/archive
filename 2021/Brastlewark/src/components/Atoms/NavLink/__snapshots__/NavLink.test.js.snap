// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavLink /> renders correctly and match snapshot 1`] = `
<a
  className="nav-link example"
  href="/"
  onClick={[Function]}
/>
`;
